internal/services/mssql/mssql_database_extended_auditing_policy_resource_test.go

```diff
@@ -409,6 +427,31 @@ resource "azurerm_mssql_database_extended_auditing_policy" "test" {
     azurerm_role_assignment.test,
   ]
 }
+
+data "azurerm_subscription" "primary" {
+}
+
+resource "azurerm_role_assignment" "test" {
+  scope                = data.azurerm_subscription.primary.id
+  role_definition_name = "Storage Blob Data Contributor"
+  principal_id         = azurerm_mssql_server.test.identity.0.principal_id
+}
+
+resource "azurerm_mssql_server_extended_auditing_policy" "test" {
+  storage_endpoint       = azurerm_storage_account.test.primary_blob_endpoint
+  server_id              = azurerm_mssql_server.test.id
+  retention_in_days      = 6
+  log_monitoring_enabled = false
+
+  storage_account_subscription_id = "1a6092a6-137e-4025-9a7c-ef77f76f2c02"
```

Comment:
this should be coming from data.TestData.SubscriptionID

Reply:
Fixed.